recipes/fmt/all/test_package/CMakeLists.txt

```diff
@@ -1,7 +1,5 @@
+cmake_minimum_required(VERSION 3.1.2)
```

Comment:
could still be 2.8.x

Reply:
According to conan documentation, `conan_basic_setup requires(TARGETS)` requires 3.1.2: https://docs.conan.io/en/latest/reference/generators/cmake.html#macros-available-in-conanbuildinfo-cmake

Reply:
Ahhh very cool, I'll have to read what that does =p